prow/cmd/sinker/main.go

```diff
@@ -124,15 +135,44 @@ func main() {
 		config:        cfg,
 	}
 
-	// Clean now and regularly from now on.
-	for {
-		start := time.Now()
-		c.clean()
-		logrus.Infof("Sync time: %v", time.Since(start))
-		if o.runOnce {
-			break
+	var wg sync.WaitGroup
+	wg.Add(1)
+
+	go func() {
```

Comment:
I don't think we need _this_ in a goroutine

Reply:
Originally, the http server was in the main thread while it is now in a goroutine.
Done.